docs/data/material/guides/nextjs/nextjs.md

```diff
@@ -0,0 +1,142 @@
+# Next.js Integration
+
+<p class="description">Learn how to use Material UI with the Next.js</p>
+
+## App Router
+
+[App router](https://nextjs.org/docs/app) is an evolution of the [Pages Router](#pages-router) and a new default way of building Next.js applications starting from version 13.
+
+This section will help you integrate Material UI with the App Router in Next.js.
+
+### Intalling the dependencies
+
+You should already have `@mui/material` and `next` installed. Then, install these dependencies:
+
+<codeblock storageKey="package-manager">
+
+```bash npm
+npm install @mui/material-nextjs @emotion/cache
+```
+
+```bash yarn
+yarn add @mui/material-nextjs @emotion/cache
+```
+
+```bash pnpm
+pnpm add @mui/material-nextjs @emotion/cache
+```
+
+</codeblock>
+
+### Configuring
+
+Inside `app/layout.tsx`, import the `AppRouterCacheProvider` and wrap all of the elements under the `<body>`:
+
+```diff
+ // app/layout.tsx
++ import { AppRouterCacheProvider } from '@mui/material-nextjs/v13-appRouter'; // or `v14-appRouter` if you are using Next.js v14
+
+ export default function RootLayout(props) {
+   const { children } = props;
+   return (
+     <html lang="en">
+       <body>
++        <AppRouterCacheProvider>{children}</AppRouterCacheProvider>
+       </body>
+     </html>
+   );
+ }
+```
+
+### Using other styling solutions
+
+If you are using a styling solution other than Emotion to customize Material UI components, set `enableCssLayer` to the `options` prop:
+
+```js
+<AppRouterCacheProvider options={{ enableCssLayer: true }}>
+```
+
+This option ensures that the styles generated by Material UI will be wrapped in a CSS `@layer mui` rule, which will be overridden by anonymous layer styles—for example, if you are customizing Material UI with CSS modules, Tailwind CSS, or even plain CSS without using `@layer`.
+
+To learn more about the CSS layer, see the [MDN CSS layer documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer).
+
+## Pages Router
+
+The steps below will help you integrate Material UI with the [Pages Router](https://nextjs.org/docs/pages/building-your-application) in Next.js for both [SSR](https://nextjs.org/docs/pages/building-your-application/rendering/server-side-rendering) (server-side rendering) and [SSG](https://nextjs.org/docs/pages/building-your-application/rendering/static-site-generation) (static site generation).
```

Comment:
```suggestion
The section below walks through the Material UI integration with the Next.js [Pages Router](https://nextjs.org/docs/pages/building-your-application), for both [Server Side Rendering](https://nextjs.org/docs/pages/building-your-application/rendering/server-side-rendering) (server-side rendering) and [Static Site Generation](https://nextjs.org/docs/pages/building-your-application/rendering/static-site-generation).
```

I think it's helpful to say what the acronym stands for in the very first time it appears on the page, regardless of more experienced devs already knowing it!